Gemfile

```diff
@@ -9,13 +9,13 @@ gem 'aws-crt' if ENV['CRT']
 gem 'http-2'
 gem 'jmespath'
 
-# json and xml parsers
+# protocol parsers
```

Comment:
Can we remove the `KITCHEN_SINK` from CI now?  I don't think its actually doing anything anymore.  Esp w/ the new approach to engines in protocol tests.

Reply:
It's necessary in the matrix as a "this is not CRT" option.